openapi/publicapi_spec.json

```diff
@@ -1211,7 +1216,7 @@
           "created_date" : {
             "type" : "string",
             "description" : "The date you created the payment.",
-            "example" : "2016-01-21T17:15:00.000Z"
+            "example" : "2016-01-21T17:15:00Z"
```

Comment:
Date formats shouldn't be changed. Existing examples in open API specs are in the correct format. See https://github.com/alphagov/pay-publicapi/pull/1577

Reply:
Yeah, not sure how this happened because I haven't touched the example values. I've pushed a change to fix this now after James pointed it out.